modules/audit-enqueued-assets/load.php

```diff
@@ -0,0 +1,312 @@
+<?php
+/**
+ * Module Name: Audit Enqueued Assets
+ * Description: Adds a CSS and JS resource checker in Site Health checks.
+ * Focus: site-health
+ * Experimental: No
+ *
+ * @package performance-lab
+ * @since 1.0.0
+ */
+
+/**
+ * Audit enqueued scripts in the frontend. Ignore /wp-includes scripts.
+ *
+ * It will save information in a transient for 12 hours.
+ *
+ * @since 1.0.0
+ */
+function perflab_aea_audit_enqueued_scripts() {
+	if ( ! is_admin() && ! get_transient( 'aea_enqueued_scripts' ) ) {
+		global $wp_scripts;
+		$enqueued_scripts = array();
+
+		foreach ( $wp_scripts->queue as $handle ) {
+			$src = $wp_scripts->registered[ $handle ]->src;
+			if ( $src && ! strpos( $src, 'wp-includes' ) ) {
+				$enqueued_scripts[] = array(
+					'src'  => $src,
+					'size' => perflab_get_resource_file_size( perflab_get_path_from_resource_url( $src ) ),
+				);
+			}
+		}
+		$expiration = apply_filters( 'perflab_aea_audit_enqueued_scripts_expiration_in_seconds', 12 * HOUR_IN_SECONDS );
+		set_transient( 'aea_enqueued_scripts', $enqueued_scripts, $expiration );
+	}
+}
+add_action( 'wp_print_scripts', 'perflab_aea_audit_enqueued_scripts' );
+
+/**
+ * Audit enqueued styles in the frontend. Ignore /wp-includes styles.
+ *
+ * It will save information in a transient for 12 hours.
+ *
+ * @since 1.0.0
+ */
+function perflab_aea_audit_enqueued_styles() {
+	if ( ! is_admin() && ! get_transient( 'aea_enqueued_styles' ) ) {
```

Comment:
Similar to my above comment, maybe this should be something closer to:
```suggestion
	if ( ! is_admin() && is_front_page() && current_user_can( 'view_site_health_checks' ) && false === get_transient( 'aea_enqueued_styles' ) ) {
```

Let's discuss on my comment above and once we've decided on something make the change in both places.